src/in/co/sdslabs/slack_lens/test/message.clj

```diff
@@ -0,0 +1,22 @@
+(ns in.co.sdslabs.slack-lens.test.message
+  (:use [org.httpkit.server])
+  (:require [compojure.route :as route]
+          [cheshire.core :as json]))
+
+(def clients (atom {}))
+
+(defn ws
+  [req]
+  (with-channel req con
+    (swap! clients assoc con true)
+          (doseq [client @clients mes (clojure.string/split-lines (slurp "messages"))]
+            (Thread/sleep 2000)(send! (key client)  mes
```

Comment:
Try some other testing strategy, `sleep` make the tests very slow and sometimes result in flaky tests.

People will start avoiding tests if they take too long